src/main/java/uk/gov/hmcts/reform/professionalapi/service/impl/OrganisationServiceImpl.java

```diff
@@ -245,6 +215,47 @@ public OrganisationsDetailResponse retrieveOrganisations() {
         return updatedOrganisationDetails;
     }
 
+    @Override
+    public OrganisationsDetailResponse retrieveAllOrganisations() {
+        List<Organisation> retrievedOrganisations = organisationRepository.findAll();
+
+        if (retrievedOrganisations.isEmpty()) {
+            throw new EmptyResultDataAccessException(1);
+        }
+
+        List<Organisation> pendingOrganisations = new ArrayList<>();
+        List<Organisation> activeOrganisations = new ArrayList<>();
+        List<Organisation> resultingOrganisations = new ArrayList<>();
+
```

Comment:
activeOrganisations list is not required remove it and activeOrganisationDetails map size